test/testdata/workflow/persistence/by_service/02-sonataflow_platform.yaml

```diff
@@ -17,6 +17,13 @@ kind: SonataFlowPlatform
 metadata:
   name: sonataflow-platform
 spec:
+  eventing:
+    broker:
+      ref:
+        apiVersion: eventing.knative.dev/v1
```

Comment:
where is this broker being created?

Reply:
It is created via make deploy-broker. We may add it to make deploy-knative if needed.

Reply:
@jianrongzhang89 if you look at the structure many of the tests, each test as an independent kustomization.yaml file that creates all the objects used by the test including the workflows.

look here:
https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/5936beebd371f3bd9e847061be0c1b9d61acf33c/test/testdata/workflow/persistence/by_service/kustomization.yaml

If you incorporate now a broker as part of the test, you must add a 0x_broker.yaml file and make it part of the kustomize.

In this way, each test execute isolated from the other. The pattern is normally like this:

1) we create a random namespace testXXX
2) we create all the objects for the test in that namespace
3) when the test finish, we delete the namespace

and so on.

This warranty that the tests executes isolated.

Reply:
Thank you @wmedvede. Done.